src/firebase/firebase.utils.js

```diff
@@ -0,0 +1,52 @@
+import firebase from "firebase/app";
+
+import 'firebase/firestore' ;
+import 'firebase/auth';
+
+const config = {
```

Comment:
Use `.env` , don't store credentials anywhere in code.

Reply:
done